test/mocha/25-http-edv-client.js

```diff
@@ -1018,9 +1019,10 @@ describe('bedrock-edv-storage HTTP API - edv-client', () => {
       } catch(e) {
         err = e;
       }
-      should.not.exist(getResult);
-      should.exist(err);
-      err.name.should.equal('NotFoundError');
+      should.exist(getResult);
+      assertNoError(err);
+      getResult.content.should.be.an('object');
+      getResult.meta.deleted.should.equal(true);
```

Comment:
I'm confused by this -- are we tracking `deleted` in multiple places -- both in `doc.deleted` and in the decrypted `meta.deleted`? If so, why both? Can we instead choose just one place? If so, which place do you think would be better and why? I could have also misread this in some way.

Reply:
Note that using `doc.deleted` crosses a trust boundary -- the server can see this. Does the server need to? For what purpose?

Reply:
Resolving this because we decided to just use `meta.deleted` and not surface a delete to the server.